src/styles/colors.constants.styles.js

```diff
@@ -0,0 +1,29 @@
+
+// color naming conventions from (some of the newer/weirder ones):
+// https://digitalsynopsis.com/design/color-thesaurus-correct-names-of-shades/
+
+export const lightBlue = '#23A5DD'
+export const blue = '#007AFF'
+export const blueDark = '#2269A3'
```

Comment:
Is this going to be the same for every project?

Reply:
these can be removed - was keeping in the off-chance we needed it for within the components - but I think best to just remove altogether - only need black and dark grey I think 
I can do a cleanup now or in a new PR?

Reply:
I think could you remove what you don't want here and push a commit?